rosidl_typesupport_opensplice_c/resource/msg__type_support_c.cpp.em

```diff
@@ -61,11 +61,6 @@ extern "C"
 {
 #endif
 
-// Forward declare the get type support function for this type.
-ROSIDL_GENERATOR_C_EXPORT_@(spec.base_type.pkg_name)
-const rosidl_message_type_support_t *
-  ROSIDL_GET_TYPE_SUPPORT_FUNCTION(@(pkg), @(subfolder), @(msg))();
-
```

Comment:
After removing this, do you not need to include the generated header that declares this function for you (with the right dll linkage)?